src/main/java/io/phasetwo/service/model/jpa/OrganizationAdapter.java

```diff
@@ -198,8 +198,7 @@ public Long getMembersCount() {
   public Stream<UserModel> getMembersStream() {
```

Comment:
@xpg do we need to update getMembersCount?

I couldn't find the usage of **/members/count** endpoint

Reply:
Yes, this should be updated and tested also.

Reply:
@kedare Please update the `/members/count` method also

Reply:
Do you know if by default it already contains the service account ? Because it's a named query that counds entries on `OrganizationMemberEntity` so I guess yes ? (if not that would mean the service accounts are on another table?)

Reply:
By default the /members/count endpoint contains both users + service account users.

Reply:
Do you know how to filter that with the named query ? It doens't looks like the `OrganizationMemberEntity` contains any information about if it's a standard user or a service account.

Reply:
You'd have to write a new named query to join on the `UserEntity`

Reply:
I guess I have to check if `getServiceAccountClientLink()` is null ?
Also I see there are no relation defined between `OrganizationMemberEntity` and `UserEntity` (just the ID field buy no FK), is there any reason for this ? Should this be added? It's not 100% clear how to join them via HQL, it looks like the usual SQL join syntax would not work on this case ?

Reply:
You can join them on the `id`

Reply:
What would be the best way to test HQL named queries ? Or I need to compile and run the whole test suite every time ?

Reply:
Maybe someone could write the HQL query directly ? I guess it would take less than a few minutes for someone that knows Hibernate well.

Reply:
Please let me know when you are able to find someone to complete this feature for you. I am closing the PR until that time.